sparkctl/cmd/list.go

```diff
@@ -50,9 +51,17 @@ func doList(crdClientset crdclientset.Interface) error {
 		return err
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "State", "Submitted age", "Completed age"})
```

Comment:
`Submission Age` and `Completion Age` sound better?

Reply:
@mrow4a can you update this PR to make the suggested fixes? Will merge after they get fixed.